### PR TITLE
Storybook - add inferno & react wrapping utils

### PR DESCRIPTION
**Pull request by pomahtri**

### CODE REVIEW

File: apps/react-storybook/stories/utils/wrappers.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    useEffect(() => {
+      componentRef.current = new component(ref.current, props);
+    }, []);
+    useEffect(() => {
+      componentRef.current.option(props)
+    });
+  
+    return <div ref={ref} />;  
+  }
+}
```
Comment (IlyaKhD) [inline apps/react-storybook/stories/utils/wrappers.tsx]: **Minor**
Add empty lime in the end of the file

**Review by IlyaKhD — DISMISSED** [review]

File: apps/react-storybook/stories/utils/wrappers.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React, { useRef, useEffect } from "react";
+import * as inferno from "inferno";
+
+export function InfernoToReact<TProps>(component) {
```
Comment (IlyaKhD) [inline apps/react-storybook/stories/utils/wrappers.tsx]: I guess lowerCamelCase is used for HOCs names in React:

```suggestion
export function asInfernoInReact<TProps>(component) {
```

or
```suggestion
export function wrapInfernoWithReact<TProps>(component) {
```

File: apps/react-storybook/stories/utils/wrappers.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        inferno.createComponentVNode(0, component, props as any),
+        ref.current
+      );
+    });
+  
+    return <div ref={ref} />;  
+  }
+}
+
+export function DxToReact<TProps>(component) {
```
Comment (IlyaKhD) [inline apps/react-storybook/stories/utils/wrappers.tsx]: Same thing, consider these renamings:

```suggestion
export function asJqInReact<TProps>(component) {
```
or
```suggestion
export function wrapJqWithReact<TProps>(component) {
```

File: apps/react-storybook/stories/utils/wrappers.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import React, { useRef, useEffect } from "react";
+import * as inferno from "inferno";
+
+export function wrapInfernoWithReact<TProps>(component) {
```
Comment (EugeniyKiyashko) [inline apps/react-storybook/stories/utils/wrappers.tsx:4]: Where is this function used?

Comment (pomahtri) [reply apps/react-storybook/stories/utils/wrappers.tsx:4]: In this PR it is not used (we don't have inferno components yet)
in CardView draft branch it is used for Card story: https://github.com/DevExpress/DevExtreme/blob/13b881b23183ee291be9a44992690de9d15f968c/apps/react-storybook/stories/card_view/Card.stories.tsx#L20

**Review by alexslavr — APPROVED** [review]

**Pull request merged**